tensorflow_cloud/run.py

```diff
@@ -28,6 +28,11 @@
 from . import validate
 
 
+def is_running_remotely():
```

Comment:
I suggest `remote()`, so users can do

`if tfc.remote():`

which is shorter and simpler

We could also have the reverse, `local()`

Reply:
Used remote.